nautobot/core/api/views.py

```diff
@@ -294,6 +296,7 @@ def get_serializer_context(self):
         return context
 
 
+# TODO: move these imports up
```

Comment:
There's an entire body of work to account for if we're going to stick with `drf-react-template stuff`. I'll make sure to revise it in my branch and get some backlog items articulated.

Reply:
OK, thanks. I'll just update the various comments related to this work to denote that.